tests/defences/preprocessor/test_spatial_smoothing_pytorch.py

```diff
@@ -24,6 +24,7 @@
 import pytest
```

Comment:
Should similar changes as in this file apply to `test_spatial_smoothing_tensorflow.py`?